source/documentation/common/database/proc-Migrating_the_Data_Warehouse_Database_to_a_Separate_Machine.adoc

```diff
@@ -8,12 +8,10 @@ To migrate the Data Warehouse service only, see xref:Migrating_the_Data_Warehous
 The new database server must have {enterprise-linux} 7 installed.
 Enable the required repositories on the new database server.
 
-ifdef::rhv-doc[]
 :context: migrate_DWH_DB
```

Comment:
Shouldn't we keep this rhv-doc only? Following link points to a file with a RHV name included

Reply:
Yes, we discussed renaming filenames to something less downstream-specific, but for now, we just use the RHV names. The content in `proc-Enabling_the_Red_Hat_Virtualization_Manager_Repositories.adoc` is however conditionalized.